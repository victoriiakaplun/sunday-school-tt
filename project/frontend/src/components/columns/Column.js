import React from 'react';

function Column({ children }) {
  return <div className="column is-half is-center">{children}</div>;
}

export default Column;
