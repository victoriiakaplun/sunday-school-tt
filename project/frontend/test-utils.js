import React from 'react'; // eslint-disable-line no-unused-vars

import { initFontAwesomeLibrary } from './src/utils';

initFontAwesomeLibrary();

// import '@testing-library/jest-dom/extend-expect';

// re-export everything
export * from '@testing-library/react';
