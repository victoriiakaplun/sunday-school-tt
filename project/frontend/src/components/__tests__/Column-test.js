import { render } from 'test-utils';

import Column from '../Column';

describe('Column', () => {
  it('renders correctly', () => {
    const { container } = render(<Column>Column entry</Column>);
    expect(container.firstChild).toMatchSnapshot(`
      <div
       class="column is-half is-center">
      Column entry
      </div>
    `);
  });
});
