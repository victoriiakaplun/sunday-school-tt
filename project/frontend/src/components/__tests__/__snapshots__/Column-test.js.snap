// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column renders correctly: 
      <div
       class="column is-half is-center">
      Column entry
      </div>
     1`] = `
<div
  class="column is-half is-center"
>
  Column entry
</div>
`;
